pylint/config/option_parser.py

```diff
@@ -17,7 +18,9 @@ def _level_options(group, outputlevel):
 
 class OptionParser(optparse.OptionParser):
     def __init__(self, option_class, *args, **kwargs):
-        optparse.OptionParser.__init__(self, option_class=Option, *args, **kwargs)
+        optparse.OptionParser.__init__(
+            self, option_class=Option, *args, version=full_version, **kwargs
```

Comment:
Maybe we could remove version entirely in OptionParse as it's handled in pre-processing?